less/quicksilver/template.less

```diff
@@ -314,3 +314,227 @@
     margin-top:-20px;
   }
 }
+
+//== Newsroom stuff
+
+//== Newsletter Edition
+
```

Comment:
Do we really need this on all pages of our site?

I would expect these to be created as separate components so that we can only build them on the newsroom website.

Reply:
Should I create a new theme for the Newsroom website?

Reply:
Yeah, that would probably make things easier!

Reply:
Ok thats what I'm going to do then.

Reply:
THanks - but please make sure to break down your CSS into smaller component files if you think some of this code could be re-used on another project.